tests/tests.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func StartServer(t *testing.T, cfg *run.Config) context.CancelFunc {
+func StartServer(t testing.TB, cfg *run.Config) context.CancelFunc {
```

Comment:
```suggestion
func StartServer(t *testing.TB, cfg *run.Config) context.CancelFunc {
```

Pointer to

Reply:
testing.TB itself is an interface while testing.T is the struct.  This is why you need pointer for testing.T while you don't need / cannot have pointer for TB

Reply:
Oops, my bad. 👍